go.mod

```diff
@@ -5,74 +5,73 @@ go 1.18
 require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/gomega v1.23.0
 	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
 	github.com/openshift/client-go v0.0.0-20220915152853-9dfefb19db2e
 	github.com/openshift/library-go v0.0.0-20220920133651-093893cf326b
-	github.com/prometheus/client_golang v1.12.2
+	github.com/prometheus/client_golang v1.14.0
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.25.1
 	k8s.io/apimachinery v0.25.1
 	k8s.io/client-go v0.25.1
 	k8s.io/klog/v2 v2.80.1
-	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
+	k8s.io/kubernetes v1.26.0
```

Comment:
Do we really want to import Kubernetes? It's a massive dependency with a long list of other dependencies which might make our own dependency management harder.

What's the diff between the util itself and the code we have? Should we just copy/paste it over as we did before?

Reply:
ack, it makes sense. I'll update the code.